python.tcc

```diff
@@ -354,11 +356,11 @@ PythonPrinter<PyV>::print(Elf::Addr remoteAddr) const {
             ssize_t fullSize;
             if (itemsize != 0) {
                 // object is a variable length object:
-                if (baseObj.ob_size > 65536 || baseObj.ob_size < 0) {
+                if (baseObj.ob_size > 65536) {
```

Comment:
Can you make this "abs(baseObj.ob_size)", so we sanity check completely huge objects?

Reply:
Fixed in most recent commit!